core-bundle/src/Cron/Cron.php

```diff
@@ -129,10 +131,13 @@ private function doRun(array $cronJobs, string $scope, bool $force = false): voi
 
         $now = new \DateTimeImmutable();
 
-        try {
-            // Lock cron table
-            $repository->lockTable();
+        $lock = $this->lockFactory->createLock(__METHOD__, 3600);
```

Comment:
Is using `__METHOD__` here best practise or commonly used? Or did you invent this?

Reply:
I claim to be the sole inventor of this practise and approve of its superior cleverness.

Reply:
> I claim to be the sole inventor of this practise
Not true! I already thought of this beforehand.